### PR TITLE
feat: Add `copyleft` icon

### PR DESCRIPTION
**Pull request by jletey**
| Preview | Composition |
| ------- | ------------- |
| Screenshot 2020-06-29 at 08 49 30 | Screenshot 2020-06-29 at 08 49 45 |

Related: #29

### CODE REVIEW

**Comment by locness3** [comment]
well nice :)

File: icons/copyleft.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="12" r="10" />
+  <path d="M9 9.35417C9.73294 8.52375 10.8053 8 12 8C14.2091 8 16 9.79086 16 12C16 14.2091 14.2091 16 12 16C10.8053 16 9.73294 15.4762 9 14.6458" />
```
Comment (ericfennis) [inline icons/copyleft.svg:13]:
```suggestion
  <path d="M9 9.35a4 4 0 1 1 0 5.3" />
```
Exported with illustrator 😋

**Comment by jletey** [comment]
Oh wow extremely simple @ericfennis Do you think we should do this with all the icons?

**Comment by ericfennis** [comment]
@jletey I think for the messy exported ones, yes.

**Comment by jletey** [comment]
@ericfennis I opened #42 😁 I'm going to go ahead and merge this.

**Pull request merged**